src/app/editor.js

```diff
@@ -23,7 +23,9 @@ var css = csjs`
     width             : 100%;
   }
 `
-document.querySelector('#editor-container').className = css['editor-container']
+setTimeout(_ => {
```

Comment:
sorry I was not precise. I meant that the code that need dom elements to be already rendered (basically the code under your `setTimeout`) could be in an `init` function that you call after the `render` function. In that way you get rid of the `setTimeout`